src/main/java/at/hannibal2/skyhanni/features/slayer/SlayerBossSpawnSoon.kt

```diff
@@ -14,7 +14,7 @@ import kotlin.time.Duration.Companion.seconds
 class SlayerBossSpawnSoon {
 
     private val config get() = SkyHanniMod.feature.slayer.slayerBossWarning
-    private val pattern = " \\(?(?<progress>[0-9.,k]+)\\/(?<total>[0-9.,k]+)\\)?.*".toPattern()
+    private val pattern = " \\(?(?<progress>[0-9.,k]+)/(?<total>[0-9.,k]+)\\)?.*".toPattern()
```

Comment:
I want to say these back slashes are still needed

Reply:
> I want to say these back slashes are still needed

You might be right - I'll revert to be safe

Reply:
Escaping the open bracket instead? Sound a bit regex behaviour manipulating

Reply:
@hannibal002 checked with nea and they said it is fine